h/templates/app.html.jinja2

```diff
@@ -18,6 +18,10 @@
               href="{{ link.href }}"/>
       {% endfor %}
     {% endif %}
+
+    <!-- Prefetch non-authenticated API routes which client will need. !-->
+    <link rel="preload" as="fetch" href="{{ request.route_url("api.index") }}" crossorigin="anonymous">
+    <link rel="preload" as="fetch" href="{{ request.route_url("api.links") }}" crossorigin="anonymous">
```

Comment:
The `crossorigin="anonymous"` attribute is needed to make the preloaded request consistent with the request the client makes when it uses `fetch(...)` to fetch this response.